scripts/test.py

```diff
@@ -110,7 +109,8 @@ def monte_carlo_sampling_scipy(N_FEATURES, SAMPLES, centered=False, strength=2,
 N_FEATURES = len(MONTE_CARLO_PYPSA_FEATURES)# only counts features when specified in config
 SAMPLES = MONTE_CARLO_OPTIONS.get("samples")   # What is the optimal sampling? Probably depend on amount of features
 SAMPLING_STRATEGY = MONTE_CARLO_OPTIONS.get("sampling_strategy")
-DISTRIBUTION = MONTE_CARLO_OPTIONS.get("distribution").title() # Change the distribution var to title case
+DISTRIBUTION = MONTE_CARLO_OPTIONS.get("distribution") # Change the distribution var to title case
+DISTRIBUTION_PARAMS = MONTE_CARLO_OPTIONS.get("distribution_params") # Change the distribution var to title case
```

Comment:
What would happen if we have requested the normal or gamma distribution, but supplied tree parameters as `distribution_params` list? 

I'd suggest to use only two first elements, but it should be documented both here and in the config and communicated during the run as a warning or even an error. What do you think?

Reply:
If we go ahead with taking just 2 parameters for the triangle distribution, we shouldnt have to worry about this.

Reply:
If we go ahead with taking just 2 parameters for the triangle distribution, we shouldnt have to worry about this.

Reply:
Though, there might be a need to handle input errors. For instance, Beta and Gamma distribution can't take numbers less than or equal to zero.